test/fixtures/detect-existing-browser-api-object/background.js

```diff
@@ -0,0 +1,15 @@
+/* global originalAPIObjects */
+
+// Register an additional message handler that always reply after
+// a small latency time.
+browser.runtime.onMessage.addListener(async (msg, sender) => {
+  if (msg !== "test-api-object-in-background-page") {
+    return false;
```

Comment:
Why `return false;`? If you don't want the handler to respond, jus return void (`return;`).
Or make this a non-async function and call `sendResponse({ ... });` in the end.